src/storage/v2/property_store.cpp

```diff
@@ -1144,6 +1144,58 @@ bool PropertyStore::SetProperty(PropertyId property, const PropertyValue &value)
   return !existed;
 }
 
+bool PropertyStore::SetProperties(std::map<storage::PropertyId, storage::PropertyValue> &properties) {
+  uint64_t size = 0;
+  uint8_t *data = nullptr;
+  std::tie(size, data) = GetSizeData(buffer_);
+  if (size != 0) {
+    return false;
+  }
+
+  uint64_t property_size = 0;
+  {
+    Writer writer;
+    for (const auto &[property, value] : properties) {
```

Comment:
I think we should filter out the properties that value is `null`. Not storing a property means it is null, thus it is unnecessarily to store null values.

Reply:
I think it is not how it works now. And how would you then know when property is null and when property just doesn't exist?

Reply:
okay, I was wrong, my apologies. If property doesn't exist we return null in such case too.

There is other problem from user side in my opinion. If property is null we don't have to store it, that is true. What if user wants some properties to be null, and for example when he wants to get all the properties stored on node there is some check on his side that number of properties stored is same as number he got. Then problem is that it will make users life worse. Right?

Reply:
@Josipmrden one client wants currently to store some null properties if I am not wrong?

Reply:
I had talk with Mrma about null, and I see what Mrma was saying, probably you think so too, from optimization side and from our schema side where we are not behaving as regular database which needs to have for each property value, and if there is no value (it is null) we don't have to store it.

Reply:
This is fixed now.